server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSSendOptionsCheckUtility.java

```diff
@@ -257,6 +268,9 @@ private void checkTextMapped(CICSParser.Cics_send_text_mappedContext ctx) {
     if (!ctx.LENGTH().isEmpty()) {
       checkHasMandatoryOptions(ctx.FROM(), ctx, "FROM");
     }
+    if (noLengthOptionsEnabled() && !ctx.FROM().isEmpty()) {
+      checkHasMandatoryOptions(ctx.LENGTH(), ctx, "LENGTH");
+    }
```

Comment:
`FROM` is already a mandatory field.
```suggestion
    if (noLengthOptionsEnabled()) {
      checkHasMandatoryOptions(ctx.LENGTH(), ctx, "LENGTH");
    }
```